compiler/luci/pass/src/RemoveUnnecessaryTransposeNetPass.cpp

```diff
@@ -40,11 +72,11 @@ class TaggedShapeAnalyzer final
                              const luci::CircleTranspose *b_tr);
 
 private:
-  void init_shape_with_tag(const luci::CircleNode *);
+  void init_shape_with_tag(const std::vector<int32_t> &);
 
-  template <loco::DataType PermType> void analyze_transpose(const luci::CircleTranspose *);
+  void analyze_transpose(const std::vector<int32_t> &);
 
-  bool analyze_reshape(const luci::CircleReshape *);
+  bool analyze_reshape(const std::vector<int32_t> &);
```

Comment:
(note) This is main changes:

* Before:
	* Each method did two works: 
	* (1) extract related information from CircleNode ptr
	* (2) update `_shape` using extracted values

* After
	* method only updates `_shape` based on the given argument 
	* The caller extracts related information and also checks the information's condition

Reply:
Also, updated methods will be used like this - https://github.com/zetwhite/ONE/blob/770025ba6fa74290dc7a5889b29f2eedceda8c36/compiler/luci/pass/src/RemoveUnnecessaryTransposeNetPass.cpp#L367-L376.